src/neptune/internal/operation_processors/operation_storage.py

```diff
@@ -27,6 +25,14 @@
 from neptune.internal.utils.logger import logger
 
 
+def get_container_dir(type_dir: str, container_id: UniqueId, container_type: ContainerType) -> str:
```

Comment:
How about adding an optional custom subdirectory (like we do in `async` & `sync` `_init_data_path`). I mean we can simplify something like:
```python
        now = datetime.now()
        container_dir = get_container_dir(ASYNC_DIRECTORY, container_id, container_type)
        data_path = f"{container_dir}/exec-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}-{os.getpid()}"
        data_path = data_path.replace(" ", "_").replace(":", ".")
        return Path(data_path)
```
with:
```python
        now = datetime.now()
        process_path = f"exec-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}-{os.getpid()}"
        process_path = process_path.replace(" ", "_").replace(":", ".")
        return get_container_dir(ASYNC_DIRECTORY, container_id, container_type, process_path)
```